### PR TITLE
[doc]update doc

### PR DESCRIPTION
**Pull request by Amber1990Zhang**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,83 +1,87 @@
...
-    |   |-- Client.py                 // the interfaces to operate nebula 
-    |   |-- ConnectionPool.py         // manger all connection, user can change the connection nums and timeout
+    |   |-- Client.py                 // interfaces controlling nebula
+    |   |-- ConnectionPool.py         // the connection pool that manages all the connections, users can specify the connection numbers and timeout when creating it
     |
-    |-- common                        // the common types
+    |-- common                        // the common data types
     |
-    |-- graph                         // the types with graphd, and the client with graphd
+    |-- graph                         // date types and client interfaces interact with graphd
```
Comment (whitewum) [inline README.md]: data

File: README.md [hunk 2]
```diff
@@ -1,83 +1,87 @@
...
+    |
+    |-- examples
+    |   |__ ClientExample.py          // the example code
     |
     |-- tests
-    |   |__ test_client.py            // the test file
+    |   |__ test_client.py            // the test code
     |
-    |-- setup.py                      // use to install or package
+    |-- setup.py                      // used to install or package
```
Comment (whitewum) [inline README.md]: use

File: README.md [hunk 3]
```diff
@@ -1,83 +1,87 @@
...
+    |   |-- ConnectionPool.py         // the connection pool that manages all the connections, users can specify the connection numbers and timeout when creating it
     |
-    |-- common                        // the common types
+    |-- common                        // the common data types
     |
-    |-- graph                         // the types with graphd, and the client with graphd
+    |-- graph                         // date types and client interfaces interact with graphd
     |
-    |-- meta                          // the types with metad, and the client with metad
+    |-- meta                          // date types and client interfaces interact with metad
```
Comment (whitewum) [inline README.md]: to interact

File: README.md [hunk 4]
```diff
@@ -1,83 +1,87 @@
...
-    |-- README.md                     // the introduce of nebula-python 
+    |-- README.md                     // the introduce of nebula-python
     |
     |__ LICENSES                      // license file
 ```
 
-## how to get the nebula-python
+## How to get nebula-python
+
+### Option One
```
Comment (whitewum) [inline README.md]: ### Option One: clone from GitHub

File: README.md [hunk 5]
```diff
@@ -1,83 +1,87 @@
...
+
+- Install
 
 ```python
 python setup.py install
 ```
 
-Method 2
-- get from pypi
+### Option Two
```
Comment (whitewum) [inline README.md]: ditto

File: README.md (hunk 2 above)
Comment (Amber1990Zhang) [reply README.md]: `setup.py` is used to install or package

**Pull request merged**